src/lib.rs

```diff
@@ -222,6 +221,37 @@ impl LogLevel for InfoLevel {
     }
 }
 
+/// Default to [`log::Level::Debug`]
+#[derive(Copy, Clone, Debug, Default)]
+pub struct DebugLevel;
+
+impl LogLevel for DebugLevel {
+    fn default() -> Option<Level> {
+        Some(Level::Debug)
+    }
+}
+
+/// Default to [`log::Level::Trace`]
+#[derive(Copy, Clone, Debug, Default)]
+pub struct TraceLevel;
+
+impl LogLevel for TraceLevel {
+    fn default() -> Option<Level> {
+        Some(Level::Trace)
+    }
+}
+
+/// Default to no logging (i.e. `None` or [`log::LevelFilter::Off`])
+#[allow(clippy::exhaustive_structs)]
```

Comment:
Why is this on this struct but none of the others?

Reply:
accidental copy paste issue - will remove.

Reply:
28a2888...31517ef